packages/dds/sequence/src/test/createSnapshotFiles.ts

```diff
@@ -4,8 +4,11 @@
  */
 
 import fs from "fs";
+import { convertSummaryTreeToITree } from "@fluidframework/runtime-utils";
 import { generateStrings, LocationBase } from "./generateSharedStrings";
 
 for (const s of generateStrings()) {
-    fs.writeFileSync(`${LocationBase}${s[0]}.json`, JSON.stringify(s[1].snapshot(), undefined, 1));
+    const summaryTree = s[1].summarize().summary;
+    const snapshotTree = convertSummaryTreeToITree(summaryTree);
```

Comment:
@anthony-murphy We need to eventually write and read the summaryTree for these tests and not convert it to ITree, right?